core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -44,6 +45,9 @@ object DbResource {
 
   val dataSource = new SlickDataSource(dataConfig)(TestExecutionContext.testExecutionContext)
   dataSource.initWithLiquibase(liquibaseChangeLog, Map.empty)
+  logger.info("executing liquibase a second time to verify changesets work on an already-initialized database ...")
+  dataSource.initWithLiquibase(liquibaseChangeLog, Map.empty)
+
```

Comment:
before this change, the `mysql-connector-java`/`liquibase-core` bug did not show up in unit tests, because unit tests always used a fresh empty db, which was initialized once with liquibase. Now, we run liquibase twice - the second liquibase run simulates what happens in prod when we redeploy Rawls on top of the existing prod db.

Reply:
I'm a little confused.  Now we're making the bug show up in tests, right?  But doesn't that mean that tests will fail?  What have we done to address the updated mysql-connector-java-liquibase unhappiness?

Reply:
the versions of `mysql-connector-java` and `liquibase-core` in this PR are compatible, so the bug will not appear. Note that this PR uses `mysql-connector-java` 8.0.22, where the original attempt used 8.0.28.